assets/scss/hero.scss

```diff
@@ -90,22 +90,74 @@
   }
 }
 
-.exhibition-page {
+.exhibition-page, .blog-post {
   background: white;
   margin-top: -3rem;
   padding-top: 0.25rem;
+  .title {
+    margin-bottom: 2rem;
+  }
   h1 {
+    font-size: 2.125rem;
     margin-top: 2.5rem;
   }
   p.lead {
     font-size: 1.5rem;
     color: $mediumgrey;
   }
+  p {
+    font-size: 1.125rem;
+  }
+  blockquote {
+    p {
+      font-size: 1.5rem;
+    }
+  }
   article, section p, section h1, h2:not(.is-size-1-5), h3, h4, h5, h6 {
     max-width: 45rem;
     margin-left: auto;
     margin-right: auto;
   }
+  article {
+    .author:last-of-type .comma {
+      display: none;
+    }
+  }
+  .article {
+    iframe {
+      border: none;
+      width: 100%;
+      min-height: 400px;
```

Comment:
this makes iframes like the audio player much too large.

If people are going to use iframes like this it's up to the editor to make sure they're styled correctly.

Just remove the height attribute.

Reply:
sure